next.config.js

```diff
@@ -1,6 +1,7 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
     reactStrictMode: true,
-    productionBrowserSourceMaps: true
+    productionBrowserSourceMaps: true,
+    output: "standalone"
```

Comment:
`output: 'standalone'` is required to build the output to a single contained image. Not sure how/if this affects the "regular" usage. From some light testing, it looks like `pnpm build && pnpm start` still works as expected, but I haven't dabbled with Next before, so who knows!